exporters/otlp/logs/src/main/java/io/opentelemetry/exporter/otlp/logs/OtlpGrpcLogExporter.java

```diff
@@ -0,0 +1,187 @@
+import io.grpc.ManagedChannel;
+import io.grpc.Status;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.BoundLongCounter;
+import io.opentelemetry.api.metrics.GlobalMeterProvider;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.exporter.otlp.internal.grpc.ManagedChannelUtil;
+import io.opentelemetry.exporter.otlp.internal.logs.LogsRequestMarshaler;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.internal.ThrottlingLogger;
+import io.opentelemetry.sdk.logging.data.LogRecord;
+import io.opentelemetry.sdk.logging.export.LogExporter;
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.ThreadSafe;
+
+/** Exports logs using OTLP via gRPC, using OpenTelemetry's protobuf model. */
+@ThreadSafe
+public final class OtlpGrpcLogExporter implements LogExporter {
+  private static final AttributeKey<String> EXPORTER_KEY = AttributeKey.stringKey("exporter");
+  private static final AttributeKey<String> SUCCESS_KEY = AttributeKey.stringKey("success");
+  private static final String EXPORTER_NAME = OtlpGrpcLogExporter.class.getSimpleName();
+  private static final Attributes EXPORTER_NAME_Attributes =
+      Attributes.of(EXPORTER_KEY, EXPORTER_NAME);
+  private static final Attributes EXPORT_SUCCESS_ATTRIBUTES =
+      Attributes.of(EXPORTER_KEY, EXPORTER_NAME, SUCCESS_KEY, "true");
+  private static final Attributes EXPORT_FAILURE_ATTRIBUTES =
+      Attributes.of(EXPORTER_KEY, EXPORTER_NAME, SUCCESS_KEY, "false");
+
+  private static final Logger internalLogger =
+      Logger.getLogger(OtlpGrpcLogExporter.class.getName());
+
+  private final ThrottlingLogger logger = new ThrottlingLogger(internalLogger);
+
+  private final MarshalerLogsServiceGrpc.LogsServiceFutureStub logsService;
+
+  private final ManagedChannel managedChannel;
+  private final long timeoutNanos;
+  private final BoundLongCounter logsSeen;
+  private final BoundLongCounter logsExportedSuccess;
+  private final BoundLongCounter logsExportedFailure;
+
+  /**
+   * Creates a new OTLP gRPC Logs Reporter with the given name, using the given channel.
+   *
+   * @param channel the channel to use when communicating with the OpenTelemetry Collector.
+   * @param timeoutNanos max waiting time for the collector to process each log batch. When set to 0
+   *     or to a negative value, the exporter will wait indefinitely.
+   * @param compressionEnabled whether or not to enable gzip compression.
+   */
+  OtlpGrpcLogExporter(ManagedChannel channel, long timeoutNanos, boolean compressionEnabled) {
+    // TODO: telemetry schema version.
+    Meter meter = GlobalMeterProvider.get().meterBuilder("io.opentelemetry.exporters.otlp").build();
```

Comment:
In general, should OTEL itself be using the global provider? I noticed this in the other exporters and was curious, since I thought we were recommending most instrumentation not use the global provider.

Reply:
We'll use this until metrics is stable and we can allow people to provide their own instance of the `MeterProvider`. This way we can at least hide any API instability, as it keeps this as an `implementation` dependency, rather than `api`.